### PR TITLE
fix a mix use of tensor( size 64x1 ) and tensor( size 64 )

### PR DESCRIPTION
**Pull request by mayorx**
fix a mix use of tensor( size 64x1 ) and tensor( size 64 ) which results:
UserWarning: Using a target size (torch.Size([64])) that is different to the input size (torch.Size([64, 1])) is deprecated. Please ensure they have the same size.

### CODE REVIEW

**Comment by wiseodd** [comment]
Hi, thanks for the pull request.
I'd be glad to merge this, but it's better to apply this fix on all pytorch scripts, not just this single script.

**Comment by mayorx** [comment]
ok, I will fix the rest pytorch scripts and test them in next several days.

**Comment by mayorx** [comment]
I already test all the pytorch scripts, and found out only two scripts(vanilla_gan, conditional_gan) had this problem. @wiseodd 

And there are two pytorch scripts (gibbsnet, dual_gan) results: UserWarning: retain_variables option is deprecated and will be removed in 0.3 ,
and one script (infogan_pytorch) results: UserWarning: Implicit dimension choice for softmax has been deprecated. 

I will fix them in other PRs.
